ts/Series/Flags/FlagsSeries.ts

```diff
@@ -251,7 +252,9 @@ class FlagsSeries extends ColumnSeries {
                     attribs.anchorX = point.anchorX;
                 }
                 graphic.attr({
-                    text: point.options.title || options.title || 'A'
+                    // Allow empty string as a flag title (#20549)
+                    text: isString(point.options.title) ? point.options.title :
+                        isString(options.title) ? options.title : 'A'
```

Comment:
The only flaw of this approach is that when setting a boolean as title, it will directly transform it to a string value, rather than falling back to the default value. If that's acceptable, then I'll make proper changes in the PR @TorsteinHonsi

Reply:
Yes, it's acceptible. I think the TS definition for this option is something like `string|undefined`, right? So if implementers set it to false they can't expect it to be hidden.